packages/turbo-ignore/src/cli.ts

```diff
@@ -30,6 +30,12 @@ turboIgnoreCli
       "The directory to run in (default: cwd)"
     )
   )
+  .addOption(
+    new Option(
+      "-b, --max-buffer <number>",
+      "maxBuffer for the child process in KB (default: 1024 * 1024)"
+    ).argParser((val) => parseInt(val, 10))
```

Comment:
If I'm reading the docs correctly, I'm pretty sure this is in bytes:

>maxBuffer [<number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) Largest amount of data in bytes allowed on stdout or stderr. If exceeded, the child process is terminated and any output is truncated. See caveat at [maxBuffer and Unicode](https://nodejs.org/api/child_process.html#maxbuffer-and-unicode). Default: 1024 * 1024

[source](https://nodejs.org/api/child_process.html#child_processexeccommand-options-callback)

```suggestion
      "maxBuffer for the child process in KB (default: 1024 KB)"
    ).argParser((val) => parseInt(val, 10) * 1024)
```